### DISCUSSION TITLE
Is it possible to rename a device?

### DISCUSSION THREAD

**Discussion by HyruleWarrior** [jertel/vuegraf#167] [Q&A]
When I first setup Emporia I put it on the 16 most important sounding circuits in my panel. Since then I have noticed that 2 of them never have any activity and aren't what I thought they were. I moved those 2 to more useful circuits and am wanting to rename those devices to something that appropriately reflects the circuit they are monitoring. I changed them in Emporia and I updated them in `vuegraf.json` but they don't seem to appear after a container restart. Do I need to drop and recreate the entire DB or is there another way I can rename these? I guess I might be able to do it in InfluxDB?

**Comment by jertel** [comment]
If you updated the vuegraf.json and restarted Vuegraf then it should be using them already. The old name will remain in InfluxDB, so if you query with a time range earlier than the rename it will continue showing the old name as well as the new name in the list of channels. You can enable verbose and/or debug logs via the command line args (`-v` and `-d`) if you want to confirm it's using the new name on new data points.

**Comment by HyruleWarrior** [comment]
Oh boy.. I had a typo in my `vuegraf.json` so Vuegraf was choking on start up... So yeah, it is now working as expected, thanks!
